shortcircuit-api/src/main/java/com/powsybl/shortcircuit/ContributionResult.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.shortcircuit;
+
+/**
+ * @author Coline Piloquet <coline.piloquet at rte-france.com>
+ */
+public class ContributionResult {
```

Comment:
Should we rename to it "feeder result" or "feeder contribution" to make explicit that it's associated to a feeder ?

Reply:
Seen together: `FeederResult` and instead of contribution, just `FeederThreePhaseFaultCurrent` for the part linked to a feeder.